eng/pipelines/azure-pipelines-codeql.yml

```
@@ -0,0 +1,58 @@
+parameters:
+- name: Codeql.TSAEnabled
+  displayName: Publish results to TSA
+  type: boolean
+  default: true
+
+variables:
+- template: eng/common-variables.yml
+  # CG is handled in the primary CI pipeline
+- name: skipComponentGovernanceDetection
+  value: true
+  # Force CodeQL enabled so it may be run on any branch
+- name: Codeql.Enabled
+  value: true
+  # Do not let CodeQL 3000 Extension gate scan frequency
+- name: Codeql.Cadence
+  value: 0
+  # Build variables
+- name: _BuildConfig
+  value: Release
+
+trigger: none
+
+schedules:
+  - cron: 0 12 * * 1
+    displayName: Weekly Monday CodeQL run
+    branches:
+      include:
+      - main
+      - release/6.0
+      - release/7.0
```

Would this work?
```suggestion
      - release/*
```

If the older release branches aren't needed, we should be able to exclude those via `exclude:` node

---

The syntax is valid, but be aware there are currently 12 branches to exclude versus only 2 to include. It seems like more work to maintain an explicit exclude list.

---

blanket release\* is always prone to unexpected runs if there is ever a test release branch ( we did create one recently) or something of sort. it would be easy to find something is not running than running more than needed. Keeping current set for now and revisit if needed.